### DISCUSSION TITLE
Exporting nvim tree output

### DISCUSSION THREAD

**Discussion by sozoalvin** [nvim-treesitter/nvim-treesitter#4313] [Q&A]
hello! searched the documentation but i believe this 'feature' may not be available yet but wanted to ask to confirm. 

are there current ways where we can export a teesitter buffer output into a text file. 

I'll like to run some codes /analysis on the buffer output to arrive at some conclusions. Such as function name length, argument counts etc.

**Comment by amaanq** [comment]
You can install the tree-sitter cli, clone the tree-sitter-language repo for the language you want, run tree-sitter generate, then tree-sitter parse <file> for that output

**Comment by clason** [comment]
or `vim.treesitter.show_tree()` on master (although the ranges are now only shown as virtual text, so you can't just yank them anymore thanks to @lewis6991)
